### ISSUE TITLE
Cannot find Jest test runner

### ISSUE THREAD

**Issue by celador** [wallabyjs/public#1833]
### Issue description or question

Running Wallaby fails due to not being able to find the test runner 
```
​​[Error] Runtime error: Failed to initialize wallaby jest.​​
​​[Error] Error: Cannot find Jest test runner​​
```

This project is using Yarn Workspaces.  Jest + Jest-CLI are both installed in the root project.  Wallaby is reading the config and grabbing the correct tests and files, but not able to find Jest. 

Jest is also installed globally at `/usr/local/bin/jest`

### Wallaby.js configuration file

```javascript
const path = require('path');
const jestPath = path.resolve(__dirname + '/node_modules/jest-cli');

console.log(jestPath);

module.exports = function(wallaby) {
  process.env.NODE_ENV = 'test'
  return {
    files: [
      'spa/App/src/javascript/**/*.js?(x|on)',
      '!spa/App/src/javascript/**/*.spec.js'
    ],
    tests: ['spa/App/src/javascript/**/*.spec.js'],
    testFramework: {
      type: 'jest',
      path: jestPath
    },
    compilers: {
      'spa/App/src/javascript/**/*.js?(x)': wallaby.compilers.babel({
        babel: require('babel-core'),
        presets: ['env', 'react'],
        plugins: ['transform-class-properties']
      })
    },
    env: {
      type: 'node',
      runner: 'node'
    },
    loose: true,
    debug: true,
    reportConsoleErrorAsError: true,
    setup: w => {
      const jestConfig = {};
      jestConfig.moduleNameMapper = {
        '^app': w.projectCacheDir + '/spa/App'
      };
      wallaby.testFramework.configure(jestConfig);
    }
  };
};

```

### Code editor or IDE name and version

Visual Studio Code v1.27.2
Wallaby-Core 1.0.149

### OS name and version

OSX Mojave

**Comment by celador** [comment]
I am using Yarn Workspaces, with one sub project using Create-React-App.  Another sub-project (the one I'm trying to test) had Jest installed.  Once I removed that dependency, tests started to work again.

**Comment by FishOrBear** [comment]
@ArtemGovorov 

`"jest": "^24.1.0"`
I have the same problem after updating to 24.1

```
module.exports = function (wallaby)
{
  return {
    files: [
      'tsconfig.json',
      'src/**/*.ts*',
      'src/**/*.tsx*',
      'src/**/*.js*'
    ],

    tests: ['__test__/**/*.test.ts*'],

    compilers: {
      '**/*.ts?(x)': wallaby.compilers.typeScript({
        module: 'commonjs',
        jsx: 'React'
      })
    },

    env: {
      type: 'node',
      runner: 'node'
    },

    testFramework: 'jest',

    debug: true
  };
};
```

**Comment by ArtemGovorov** [comment]
@FishOrBear What is your Wallaby Core version (printed in Wallaby Console when you start it)?

**Comment by FishOrBear** [comment]
@ArtemGovorov  [Info]​​​​​ Started Wallaby.js Core v1.0.634

VsCode 1.31

**Comment by ArtemGovorov** [comment]
@FishOrBear Your license had expired prior to us adding support for Jest 24 (that had introduced a few breaking changes that we needed to address). Current version of Wallaby Core (v1.0.652) fully supports Jest 24 (including 24.1), so you may either
- use an older version of Jest supported by your current license consider,
- or [renew your current license with a discount](https://wallabyjs.com/purchase/#upgrade) to get the version of Wallaby that supports Jest 24 (and 12 months of upgrades and new features).

**Comment by smeijer** [comment]
@ArtemGovorov, having the same issue in an empty project. Any ideas?

```shell
wallaby.js started
core v1.0.659
```

package.json
```json
{
  "name": "empty-project",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "dependencies": {
    "@babel/core": "^7.3.4",
    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
    "@babel/preset-env": "^7.3.4",
    "@babel/register": "^7.0.0",
  },
  "devDependencies": {
    "jest": "^24.5.0"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC"
}
```

wallaby.config.js
```js
module.exports = function () {
  return {
    files: ['src/**/*.js', '!src/**/*.tests.js'],

    tests: ['src/**/*.tests.js'],

    env: {
      type: 'node',
      runner: 'node'
    },

    testFramework: 'jest'
  };
};
```

Output:

```
Failed to initialize wallaby jest.
Error: Cannot find Jest test runner
    at Module._compile (module.js:652:30)
```

**Comment by smcenlly** [comment]
We tested your package.json and wallaby config and it's working for us on the latest version of Wallaby (core v1.0.665).

Jest 24.3.1 introduced breaking changes that we have fixed in Wallaby core v1.0.660. You will need to update your core version. If it doesn't automatically update, you can [force the update](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

If your license has expired and you are running an older version of Wallaby, you may downgrade to an earlier version of Jest or else renew your Wallaby license.

**Comment by smeijer** [comment]
Got it! I thought I was on the safe, due to the comment above me:

> Current version of Wallaby Core (v1.0.652) fully supports Jest 24 (including 24.1), so you may either

License extended (expired few days ago), version upgraded, and problem solved.

Thanks for the fast response @smcenlly. Nice to see some green boxes again.
